common-libs/common/src/Affinity.ts

```diff
@@ -86,8 +86,14 @@ export class Affinity {
       throw new Error('Signature on token is invalid')
     }
 
-    if (payload.exp < Date.now()) {
-      throw new Error('Token expired')
+    if (payload.expiresAt) {
+      if (payload.expiresAt < Date.now()) {
+        throw new Error('Token expired')
+      }
+    } else {
```

Comment:
```suggestion
    } else if (payload.exp) {
```

Is it possible to have no expiration field?